<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>kthemeui</class>
 <widget class="QDialog" name="kthemeui">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1266</width>
    <height>591</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>12</red>
        <green>45</green>
        <blue>87</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>12</red>
        <green>45</green>
        <blue>87</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>12</red>
        <green>45</green>
        <blue>87</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>12</red>
        <green>45</green>
        <blue>87</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="ktheme1btn">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>220</y>
     <width>301</width>
     <height>291</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="ktheme2btn">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>220</y>
     <width>301</width>
     <height>291</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="ktheme3btn">
   <property name="geometry">
    <rect>
     <x>1030</x>
     <y>220</y>
     <width>301</width>
     <height>291</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>50</y>
     <width>321</width>
     <height>31</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>254</red>
         <green>103</green>
         <blue>54</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="WindowText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>254</red>
         <green>103</green>
         <blue>54</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled/>
    </palette>
   </property>
   <property name="font">
    <font>
     <family>Britannic Bold</family>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="text">
    <string>KITCHEN - THEME</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
